### PR TITLE
Changed license in source/ to MIT

### PR DESCRIPTION
**Pull request by marcusnaslund**
This is going to be an enormous PR.

Part of #1127

### CODE REVIEW

File: source/draw/gpu/GpuImage.ooc [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
```
Comment (marcusnaslund) [inline source/draw/gpu/GpuImage.ooc]: @fredrikbryntesson @simonmika Is this OK? Do we need the whole thing in each file?

Comment (fredrikbryntesson) [inline source/draw/gpu/GpuImage.ooc]: We do not need it in every file to make the license valid, but it is maybe a good idea to make it clear which license the repository uses.

**Comment by marcusnaslund** [comment]
Ignore my python script, it won't be merged into upstream.

**Comment by simonmika** [comment]
Do it in every ooc-file. Enable a test on the ci-server to verify that every file has the license.

**Comment by marcusnaslund** [comment]
Good idea.

File: LICENSE [hunk 2]
```diff
@@ -1,166 +1,21 @@
...
-
-  If the Library as you received it specifies that a proxy can decide
-whether future versions of the GNU Lesser General Public License shall
-apply, that proxy's public statement of acceptance of any version is
-permanent authorization for you to choose that version for the
-Library.
-
+The MIT License (MIT)
+
+Copyright (c) 2015 magic-lang
```
Comment (marcusnaslund) [inline LICENSE]: Oops, wrong year.

**Comment by simonmika** [comment]
The year should be the year when the file was first created and additional years or ranges should be added on major updates.

File: source/draw/DrawState.ooc [hunk 3]
```diff
@@ -1,18 +1,10 @@
...
-//
-// You should have received a copy of the GNU Lesser General Public License
-// along with this program. If not, see <http://www.gnu.org/licenses/>.
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
```
Comment (fredrikbryntesson) [inline source/draw/DrawState.ooc]: Empty line

File: source/net/utilities.ooc [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
 	between UDPSocket and ServerSocket.
 */
```
Comment (fredrikbryntesson) [inline source/net/utilities.ooc]: Huh?

Comment (marcusnaslund) [inline source/net/utilities.ooc]: Oops. Fixed.

**Pull request merged**